odoo_sh/getting_started/online-editor.rst

```diff
@@ -0,0 +1,195 @@
+without having to restart the server manually.
+
+.. image:: ./media/interface-editor-automaticreload.gif
+   :align: center
+
+However, if the change is a data stored in database, such as the label of a field, or a view,
+you have to update the according module to apply the change.
+You can update the module of the currently opened file by using the menu
+:menuselection:`Odoo --> Update current module`. Note that the file considered as currently opened
+is the file focused in the text editor, not the file highlighted in the file browser.
+
+.. image:: ./media/interface-editor-update-current-module.png
+   :align: center
+
+You can also open a terminal and execute the command:
+
+.. code-block:: bash
+
+  $ odoo-bin -u <comma-separated module names> --stop-after-init
+
+.. _odoosh-gettingstarted-online-editor-push:
+
+Commit & Push your changes
+==========================
+
+You have the possibility to commit and push your changes to your Github repository.
+
+* Open a terminal (:menuselection:`File --> New --> Terminal`),
+* Change the directory to *~/src/user* using :code:`cd ~/src/user`,
+* Stage your changes using :code:`git add`,
+* Commit your changes using :code:`git commit`,
+* Push your changes using :code:`git push https HEAD:<branch>`.
+
+In this last command,
+
+* *https* is the name of your *HTTPS* Github remote repository
+  (e.g. https://github.com/username/repository.git),
+* HEAD is the reference to the latest revision you committed,
+* <branch> must be replaced by the name of the branch to which you want to push the changes,
+  most-likely the current branch if you work in a development build.
+
+.. image:: ./media/interface-editor-commit-push.png
+   :align: center
+
+.. Note::
+  The SSH Github remote is not used because your SSH private key
+  is not hosted in your build containers (for obvious security concerns)
+  nor forwarded through an SSH Agent (as you access this editor through a web browser)
+  and you therefore cannot authenticate yourself to Github using SSH.
+  You have to use the HTTPS remote of your Github repository to push your changes,
+  which is added automatically named as *https* in your git remotes.
+  You will be prompted to enter your Github username and password.
+  If you activated the two-factor authentication on Github,
+  you can create a
+  `personal access token <https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/>`_
+  and use it as password.
+
+
+.. Note::
+  The Git source folder *~/src/user* is not checked out on a branch but rather on a detached revision:
+  This is because builds work on specific revisions rather than branches.
+  In other words, this means you can have multiple builds on the same branch, but on different revisions.
+
+Once your changes pushed,
+according to your :ref:`branch push behavior <odoosh-gettingstarted-branches-tabs-settings>`,
+a new build may be created. You can continue to work in the the current editor you pushed from,
```

Comment:
You can continue to work in **the** (1x, + je virerais le current) editor you pushed from, as it will have the same revision **as** the new build **that was created**, ...